Makefile

```diff
@@ -0,0 +1,42 @@
+FMT_OPTS = -co -XTypeApplications -o -XImportQualifiedPost
+FIND_OPTS = src test -type f -name '*.hs'
+
+build:
+	cabal build
+
+test:
+	cabal test
+
+# Dogfood the dev version
+analyze:
+	cabal run fossa -- analyze --output --debug --filter 'cabal@./'
+
+# Copy the built binary into the local root
+fossa:
+	cp $(shell cabal list-bin fossa) ./fossa
+
+install-local: fossa
+
+# Run analysis on the sandbox directory
+sandbox: fossa
+	./fossa analyze --output --debug --record ./sandbox > fossa.json
+
+# Run the sandbox with replay mode on
+replay: fossa fossa.debug.json
+	./fossa analyze --output --debug --replay fossa.debug.json ./sandbox > fossa.replay.json
```

Comment:
This is interesting.

I'm not sure this is _useful_ in general. You can't replay a customer debug log unless you use the same folder name, and there's no point in replaying your own `./sandbox` run because you could just re-run `make build && make sandbox` since you'd have to have the project files locally to generate the record in the first place.

Unless I'm missing something, this doesn't seem useful. I'd remove it.

Reply:
Yeah, I forgot about the working directory part when writing this.